### PR TITLE
Create 1.mdx

### PR DESCRIPTION
**Pull request by webdevKmer**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Closes # 
- What does this PR change? Give us a brief description.
A translation of the page from English to French.

### CODE REVIEW

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+i18nReady: false
+---
+import Badge from '~/components/Badge.astro';  
+import Checklist from '~/components/Checklist.astro';  
+import Box from '~/components/tutorial/Box.astro';
+
+
+## De quoi ai-je besoin pour commencer ?
+
+Si vous avez des connaissances en **HTML**, **Markdown**, **CSS**, et si vous avez un peu de connaissances en **JavaScript**, alors vous etes prets a debuter! Vous pourrez etre en mesure de terminer le tutoriel juste en suivant les instructions. Astro est pour tout le monde! 🧑‍🚀 👩‍🚀 👨‍🚀
```
Comment (thomasbnt) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:14]: Hello! Il manque quelques accents comme pour "etes" "prets" "a" "debuter"

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Si vous avez des connaissances en **HTML**, **Markdown**, **CSS**, et si vous avez un peu de connaissances en **JavaScript**, alors vous etes prets a debuter! Vous pourrez etre en mesure de terminer le tutoriel juste en suivant les instructions. Astro est pour tout le monde! 🧑‍🚀 👩‍🚀 👨‍🚀 
+
+Vous devrez aussi posseder un compte [GitHub](https://github.com) (ou similaire) pour publier votre projet sur le web.
+
+<details>
+<summary>Comment est ce que j'utilise les cases a cocher en bas de chaque page?</summary>
+
+Vous les cochez!
+ 
+A la fin de chaque page, vous trouverez une liste de cases a cocher, de taches que vous etes deja en mesure de faire. Cocher les cases correspondantes pour visualiser votre progres sur le trackeur du tutoriel
```
Comment (thomasbnt) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:23]: Même issue d'orthographe "taches", "deja" "progres" et le point à la fin

**Review by thomasbnt — COMMENTED** [review]
Hello superbe merci ! 
Il manque pas mal d'accents sur le fichier, et revoir la ponctuation sur certaines phrases!

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+<details>
+<summary>Que faire si j'ai besoin d'aide ou si je veux en apprendre plus sur Astro?</summary>
+
+Notre [amical serveur Discord Astro](https://astro.build/chat) est le lieu ou il faut etre! 
+
+Entrer dans le canal de support forum et posez vos questions, dites `bonjour` et chattez sur le canal `#general` ou `#off-topic`.
+</details>
+
+<details>
+<summary>Ou juis-je laisser un feedback a propos de ce tutoriel?</summary>
```
Comment (dreyfus92) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:47]:
```suggestion
<summary>Où puis-je laisser un feedback à propos de ce tutoriel ?</summary>
```

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Ce tutoriel est un projet de notre equipe Docs. Vous pouvez nous trouver sur Discord sur le canal `#docs`, or nous envoyer vos difficultes sur [Repo sur GitHub](https://github.com/withastro/docs/issues). 
+</details>
+
+<Box icon="check-list">
+
+## Case a cocher pour continuer
+
+<Checklist>
+- [ ] Je suis pret a construire cala.!
```
Comment (dreyfus92) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:57]:
```suggestion
- [ ] Je suis prêt à construire cela !
```

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Si vous [creer un nouveau, projet Astro vide](/en/install/auto/) et etes a l'aise avec votre setup, libre a vous de continuer directement au [Module 2](/en/tutorial/2-pages/) ou vous comencerez a creer de nouvelles pages dans votre projet.
+</details>
+
+<details>
+<summary>Que faire si j'ai besoin d'aide ou si je veux en apprendre plus sur Astro?</summary>
+
+Notre [amical serveur Discord Astro](https://astro.build/chat) est le lieu ou il faut etre! 
+
+Entrer dans le canal de support forum et posez vos questions, dites `bonjour` et chattez sur le canal `#general` ou `#off-topic`.
```
Comment (dreyfus92) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:43]:
```suggestion
Rejoignez dans le canal de support forum et posez vos questions, dites `bonjour` et chattez sur le canal `#general` ou `#off-topic`.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Salut! bon travail, je vous laisse quelques suggestions 😁

File: src/content/docs/fr/tutorial/0-introduction/1.mdx (hunk 1 above)
Comment (webdevKmer) [reply src/content/docs/fr/tutorial/0-introduction/1.mdx:14]: souci de clavier... Je corrige cela des que possible

File: src/content/docs/fr/tutorial/0-introduction/1.mdx (hunk 2 above)
Comment (webdevKmer) [reply src/content/docs/fr/tutorial/0-introduction/1.mdx:23]: bien recu

**Comment by Yan-Thomas** [comment]
Thanks @webdevKmer, I guess this looks good to go, I see @thomasbnt pointed out a few accentuation issues, were these addressed yet? If yes, I'l proceed with the merge.

**Comment by thomasbnt** [comment]
Bonjour @webdevKmer, des nouvelles concernant ce PR? 🙌

**Review by pioupia — CHANGES_REQUESTED** [review]
Accent and French mistakes to correct.

File: src/content/docs/fr/tutorial/0-introduction/1.mdx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+(Cette information est uniquement sauvegardée dans le stockage local de votre naviguateur, et n'est disponible nulle part ailleurs. Aucune information n'est envoyée ou sauvegardée par Astro.)
+</details>
+
+<details>
+<summary> Le module 1 est constitué de sujets que je connais déjà. Puis-je les passer ?</summary>
+
+Vous pouvez utiliser les leçons du [Module 1](/fr/tutorial/1-setup/) pour vous assurer que vous disposez des outils de développement et des comptes en ligne dont vous aurez besoin pour terminer le tutoriel. Il vous guidera dans la création d'un nouveau projet Astro, son stockage sur GitHub et son déploiement sur Netlify.
+
+Si vous [créez un nouveau projet Astro vide](/en/install/auto/) et êtes à l'aise avec votre setup, libre à vous de continuer directement au [Module 2](/en/tutorial/2-pages/) où vous commencerez à créer de nouvelles pages dans votre projet.
```
Comment (delucis) [inline src/content/docs/fr/tutorial/0-introduction/1.mdx:35]: Mistake in the link here:
```suggestion
Si vous [créez un nouveau projet Astro vide](/fr/install/auto/) et êtes à l'aise avec votre setup, libre à vous de continuer directement au [Module 2](/fr/tutorial/2-pages/) où vous commencerez à créer de nouvelles pages dans votre projet.
```

**Pull request merged**